daemon/docker.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/rancher/convoy/api"
 	. "github.com/rancher/convoy/convoydriver"
 	"github.com/rancher/convoy/util"
+	"regexp"
```

Comment:
This is purely organizational/stylistic, but I've found that organizing all builtin imports up top and then keeping 3rd party ones below makes it easier to quickly assess what resources a given file is using.

So in this case I suggest moving these imports up top.